examples/website/src/shared/route/articleTemplateContainer.tsx

```diff
@@ -419,7 +419,7 @@ export function PeerProfileBlock({peer, article}: PeerProfileBlockProps) {
         <div className={css(PeerProfileFiller)} />
       </div>
       <div className={css(PeerProfileCallToActionURL)}>
-        {peer?.callToActionText?.length && peer?.callToActionText?.length > 0 ? (
+        {peer?.callToActionText?.length ? (
```

Comment:
both a `string` and an `array` have the property `length`. Maybe use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray